tests/test_parsing.py

```diff
@@ -64,6 +65,43 @@ def test_call_parsing_errors(self):
                     parse_call(input_arg)
                 self.assertEqual(expected_message, e.exception.message)
 
+    def test_url_conversion_public(self):
```

Comment:
:D Tests. For completness a couple of tests of it not parsing?

e.g.
- "https://complib.org/method/12834"
- ""
- "not a number"

Reply:
Yep I forgot about that :sweat_smile:.  I should also decide on what error messages should be produced in certain cases (like loading a method instead of a comp).

Reply:
Done!